walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -19,24 +19,26 @@ const STRINGS = AppStrings['common'];
  * @return {Object} the requirements list in a key-value mapping format.
  */
 function getRequirementsList() {
-    // TODO(issue/17): GET request to servlet to get from database
-    // returning some hardcoded values for now
-    return {
-      'O_LEVEL': 'O Level',
-      'LANGUAGE_ENGLISH': 'English',
-      'DRIVING_LICENSE_C': 'Category C Driving License',
-    };
+  // TODO(issue/17): GET request to servlet to get from database
+  // returning some hardcoded values for now
+  return {
+    'O_LEVEL': 'O Level',
+    'LANGUAGE_ENGLISH': 'English',
+    'DRIVING_LICENSE_C': 'Category C Driving License',
+  };
 }
 
 /**
  * Sets the error message according to the param.
- * @param {String} errorMessageElementId the element id for html error message div.
+ * @param {String} errorMessageElementId the element id
+ * for html error message div.
  * @param {String} msg the message that the error div should display.
- * @param {boolean} includesDefault whether the includes default message. Default to be true.
- * message should be included.
+ * @param {boolean} includesDefault whether the includes default message.
+ * Default to be true.
```

Comment:
super nit: i think in terms of formatting this should be attached to the `@param`. So if you format automatically it should go to the new line with some space.